src/appkit.cs

```diff
@@ -3891,6 +3973,21 @@ public interface NSColorSpace : NSCoding, NSSecureCoding {
 		[Export ("availableColorSpacesWithModel:")]
 		NSColorSpace [] AvailableColorSpacesWithModel (NSColorSpaceModel model);
 
+		[Mac (10, 12)]
+		[Static]
+		[Export ("extendedSRGBColorSpace")]
+		NSColorSpace ExtendedSRGBColorSpace { get; }
```

Comment:
not sure, IIRC it is named `sRGB` so maybe `sRgb` is alright maybe someone else has a better idea 😄